phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixResultSet.java

```diff
@@ -213,6 +216,13 @@ public void close() throws SQLException {
             statement.getResultSets().remove(this);
             overAllQueryMetrics.endQuery();
             overAllQueryMetrics.stopResultSetWatch();
+            if (context.getCurrentTable() != null &&
+                    context.getCurrentTable().getTable() != null) {
```

Comment:
you might want to check  context.getCurrentTable().getTable().getPhysicalName().toString() is not null

Reply:
Added the string.utils check